src/components/BlogEntries.js

```diff
@@ -30,9 +31,9 @@ class BlogEntries extends Component {
           {(blogs && this.state.postCheck === true) || this.state.newsCheck === true ? (
             blogs.map(blog => {
               if (blog.entry.entryType === 'post' && this.state.postCheck === true) {
-                return <Post key={blog.entry.uuid} post={blog} />;
+                return <Post key={blog.entry.uuid} post={blog} auth={auth} profile={profile} users={users} />;
```

Comment:
If `Post` and `News` only use `auth.uid`, then only pass that. As a rule: pass only what is needed, don't let the receiver have to pick what it needs.